code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -219,10 +221,14 @@
 
 	if(src.loc == W)
 		return 0 //Means the item is already in the storage item
-	if(contents.len >= storage_slots)
+	if(contents.len >= storage_slots && storage_slots > 0)
 		if(!stop_messages)
 			to_chat(usr, "<span class='notice'>\The [src] is full, make some space.</span>")
 		return 0 //Storage item is full
+	if(contents.len >= storage_slots && storage_slots <= 0)
+		if(!stop_messages)
+			to_chat(usr, "<span class='notice'>\The feel around on the [src], but it doesn't have storage!</span>")
```

Comment:
???

Reply:
If the red_ribbon_arm were to be a storage_slots=0 belt, it would need to do something in regards to the show_to proc being disabled, like telling the user that this storage item has no storage.

Reply:
I think what he means is `\The feel`